ui/server/src/test/scala/pl/touk/nussknacker/ui/process/repository/DBFetchingProcessRepositorySpec.scala

```diff
@@ -201,29 +200,29 @@ class DBFetchingProcessRepositorySpec
     val latestDetails = fetchLatestProcessDetails(processName)
     latestDetails.processVersionId shouldBe VersionId.initialVersionId
 
-    updateProcess(latestDetails.processId, GraphProcess.empty, false).newVersion.get shouldBe VersionId(2)
+    updateProcess(latestDetails.processId, ProcessCanonizer.canonize(ProcessTestData.validProcess), false).newVersion.get shouldBe VersionId(2)
```

Comment:
Maybe at ProcessTestData we can do validCanonicedProcess? ;)

Reply:
I will do some transition methods between dsl model and canonicalprocess - most of this `ProcessCanonizer.canonize` will disappear. I left it on next change because it will cause moving ProcessCanonizer between modules which will be not straightforward